unit_tests/test_cassandra_stress_thread.py

```diff
@@ -84,9 +84,10 @@ def test_03_cassandra_stress_client_encrypt(request, docker_scylla, params):
     loader_set = LocalLoaderSetDummy()
 
     cmd = (
-        """cassandra-stress write cl=ONE duration=1m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=1) """
-        """compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native """
-        """-rate threads=10 -pop seq=1..10000000 -log interval=5"""
+        "cassandra-stress write cl=ONE duration=1m -schema 'compaction(strategy=SizeTieredCompactionStrategy) "
+        "replication(strategy=NetworkTopologyStrategy,replication_factor=1)' -mode cql3 native -rate threads=10 "
+        "-pop seq=1..10000000 -log interval=5 "
+        "-transport 'truststore=/etc/scylla/ssl_conf/truststore.jks truststore-password=cassandra'"
```

Comment:
Check if CassandraStressThread adds `-transport` parameter properly, when is instantiated in unit test (it should) and remove this hardcoded parameter.